src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx

```diff
@@ -17,7 +17,7 @@ import styled from "styled-components"
 import { Media } from "v2/Utils/Responsive"
 
 // Disables arrows in numeric inputs
-const NumericInput = styled(LabeledInput).attrs({ type: "number" })`
+export const NumericInput = styled(LabeledInput).attrs({ type: "number" })`
```

Comment:
with TS i feel that stuff like this (cross app/component exports and imports) are better than ad hoc shared util/ folders and what not. at least until another use possibly.